charts/argo-rollouts/templates/controller/servicemonitor.yaml

```
@@ -17,6 +17,11 @@ metadata:
 spec:
   endpoints:
   - port: {{ .Values.controller.metrics.service.portName }}
+    {{- if .Values.controller.metrics.serviceMonitor.tlsConfig }}
+    scheme: https
+    tlsConfig:
+      {{- toYaml .Values.controller.metrics.serviceMonitor.tlsConfig | nindent 6 }}
+    {{- end }}
```

nits; `with` is simpler ? 😄 
```suggestion
    {{- with .Values.controller.metrics.serviceMonitor.tlsConfig }}
    scheme: https
    tlsConfig:
      {{- toYaml . | nindent 6 }}
    {{- end }}
```